apps/test/solutions/maze/3_1.js

```diff
@@ -0,0 +1,37 @@
+var testUtils = require('../../util/testUtils');
+var TestResults = require(testUtils.buildPath('constants.js')).TestResults;
+
+var reqBlocks = function () {
+  // stick this inside a function so that it's only loaded when needed
+  return require('../../util/testUtils').requireWithGlobalsCheckBuildFolder('maze/requiredBlocks.js');
+};
+
+module.exports = {
+  app: "maze",
+  levelFile: "levels",
+  levelId: "3_1",
+  tests: [
+    {
+      description: "Verify solution",
+      editCode: true,
+      timeout: 15000,
+      expected: {
+        result: true,
+        testResult: TestResults.ALL_PASS
+      },
+      // missingBlocks: [],
+      xml: 'moveForward();moveForward();moveForward();'
```

Comment:
I wonder if we should change this name (xml) to something more accurate now that it can be xml or JS. Not something you need to change for this PR in any case.